genome_kit/gk_data.py

```diff
@@ -56,6 +56,17 @@ def get_file(filename):
 def upload_file(filepath:str, filename:str, metadata:Dict[str, str]=None):  # pragma: no cover
     """compatibility wrapper for :py:meth:`~genome_kit.DataManager.upload_file`"""
 
+    chrom_sizes_ext = ".chrom.sizes"
```

Comment:
I know I was the one that recommended this, but is there a way we can support offline users as well (for instance, that uses the starter script)

Reply:
I'll add hash files to the starter script.